src/handlers/K8sCleanupActionHandler.ts

```diff
@@ -25,13 +25,18 @@ export class K8sCleanupActionHandler extends ActionHandler {
         .keys({
             dryRun: Joi.boolean().default(false),
             namespace: Joi.string().min(1).required(),
-            allowed: Joi.object({
-                storageClasses: Joi.array().min(1).items(Joi.string().required()),
-                persistentVolumeClaims: Joi.array().min(1).items(Joi.string().required()),
-                helms: Joi.array().min(1).items(Joi.string().required()),
-                secrets: Joi.array().min(1).items(Joi.string().required()),
-                configMaps: Joi.array().min(1).items(Joi.string().required()),
-            }),
+            abstractions: Joi.array()
+                .min(1)
+                .items(Joi.string().required())
+            ,
+            allowed: Joi.object()
+                .unknown(true)
```

Comment:
With this you allowed any other names that do not match the pattern below with any kind values. I assume that .unknown directive should just be removed